README.md

```diff
@@ -1,3 +1,25 @@
-# harbor
+# Harbor
+
+A chrome extension for Cyberoam authentication with functionalities of login/logout, tracking data usage with a easy to use interface providing a smooth user experience.
+
+## Contributing
+
+- Fork this repo and then clone the repo with `git clone https://github.com/osdc/harbor.git` or you can use ssh too.
+
+- Now open chrome, naviagate to **Options** > **More tools** > **Extensions**
+
+- Click on load unpacked and select harbor. Voila your extension is ready!
+
+- In case of any dilemmas you can create an issue or reach us at our [slack channel](https://jiit-lug.slack.com/). 
```

Comment:
the given link will ask people to sign up, you can use [this](http://bit.ly/jiit-lug) link to directly add them in the channel